docs/design/gap-jumping.md

```diff
@@ -121,10 +121,10 @@ in the buffered ranges is caused by gaps in the content, not by content that
 will be available later.
 
 We already fetch the segment before the target time when seeking.  We also only
-adjust the end time of a segment, so the start time is “close” to the real time.
+adjust the end time of a segment, so the start time is “close” to the real-time.
 This means that when we seek, there will always be content buffered before the
-playhead.  Once we have buffered past the playhead, we will know if we have
-seeked into a gap.
+playhead. Once we have buffered past the playhead, we will know if we have
+sought into a gap.
```

Comment:
Haha. Yeah, "seeked" isn't a real word, but I'm not sure if people will understand that "sought" is meant to be the past tense of "seek", the video playback term.